tests/libtest/lib3026.c

```diff
@@ -26,12 +26,70 @@
 #include "testutil.h"
 #include "warnless.h"
 
-#ifdef HAVE_PTHREAD_H
+#define NUM_THREADS 1000
+
+#ifdef WIN32
+static DWORD WINAPI run_thread(LPVOID ptr)
+{
+  CURLcode *result = ptr;
+
+  *result = curl_global_init(CURL_GLOBAL_ALL);
+  if(*result == CURLE_OK)
+    curl_global_cleanup();
+
+  return 0;
+}
+
+int test(char *URL)
+{
+  CURLcode results[NUM_THREADS];
+  HANDLE ths[NUM_THREADS];
+  unsigned tid_count = NUM_THREADS, i;
+  int test_failure = 0;
+  curl_version_info_data *ver;
+  HANDLE th;
+  (void) URL;
+
+  ver = curl_version_info(CURLVERSION_NOW);
+  if((ver->features & CURL_VERSION_THREADSAFE) == 0) {
+    fprintf(stderr, "%s:%d On Windows but the "
+            "CURL_VERSION_THREADSAFE feature flag is not set\n",
```

Comment:
Is this really an error, isn't it expected when target is < 0x600?

Reply:
Yes, but I think the test should not be executed at all in that case.

Reply:
I don't think we have a very convenient way to disable a test for that specific condition though. It can be made as a `<precheck>` I suppose.

Reply:
Hm, can't I detect the feature not being present via the test data file?

Reply:
Thanks @jay